compose.yaml

```
@@ -0,0 +1,326 @@
+# Local development-only compose file. This is not suitable for production!
+
+x-janus-common: &janus_common
+  image: us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_aggregator:0.7.7
+  restart: always
+  healthcheck:
+    test: ["CMD", "wget", "http://localhost:8000/healthz"]
+  depends_on:
+    janus_1_migrate:
+      condition: service_completed_successfully
+    janus_2_migrate:
+      condition: service_completed_successfully
+
+x-janus-migrate: &janus_migrate
+  image: us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_db_migrator:0.7.7
+  command:
+    - migrate
+    - run
+    - --source
+    - /migrations
+    - --connect-timeout
+    - "60"
+  depends_on:
+    postgres:
+      condition: service_started
+
+x-janus-environment: &janus_environment
+  RUST_LOG: info
+  PGPASSWORD: postgres
+  DATASTORE_KEYS: 1B6szboUUtMfyrLsIVE20g
+  AGGREGATOR_API_AUTH_TOKENS: "0000"
+
+services:
+  postgres:
+    image: docker.io/library/postgres:16
+    ports:
+      - "5432:5432"
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: postgres
+    configs:
+      - source: postgres_init
+        target: /docker-entrypoint-initdb.d/postgres_init.sql
+
+  divviup_api_migrate:
+    build:
+      context: .
+      dockerfile: Dockerfile.fast
+      args:
+        # This isn't strictly required for migrations, but prevents docker from needlessly rebuilding
+        # the image.
```

Suggested rephrasing for clarity:
```suggestion
        # This isn't strictly required for migrations, but it allows reusing one
        # container image for both this and the divviup_api service.
```